manim/mobject/mobject.py

```diff
@@ -2040,40 +2041,229 @@ def construct(self):
 
-    def arrange_in_grid(self, n_rows=None, n_cols=None, **kwargs):
-        """Sorts :class:`~.Mobject` next to each other on screen using a grid.
+    def arrange_in_grid(
+        self,
+        rows: Optional[int] = None,
+        cols: Optional[int] = None,
+        buff: Union[float, Tuple[float, float]] = MED_SMALL_BUFF,
+        cell_alignment: np.ndarray = ORIGIN,
+        row_alignments: Optional[str] = None,  # "ucd"
+        col_alignments: Optional[str] = None,  # "lcr"
+        row_heights: Optional[Iterable[Optional[float]]] = None,
+        col_widths: Optional[Iterable[Optional[float]]] = None,
+        flow_order: str = "rd",
+        **kwargs,
+    ) -> "Mobject":
+        """Arrange submobjects in a grid.
+
+        Parameters
+        ----------
+        rows
+            The number of rows in the grid.
+        cols
+            The number of columns in the grid.
+        buff
+            The gap between grid cells. To specify a different buffer in the horizontal and
+            vertical directions, a tuple of two values can be given - `(row, col)`.
+        cell_alignment
+            The way each submobject is aligned in its grid cell.
+        row_alignments
+            The vertical alignment for each row (top to bottom). Accepts the following characters: `"u"` -
+            up, `"c"` - center, `"d"` - down.
+        col_alignments
+            The horizontal alignment for each column (left to right). Accepts the following characters `"l"` - left,
+            `"c"` - center, `"r"` - right.
+        row_heights
+            Defines a list of heights for certain rows (top to bottom). If the list contains `None`, the corresponding row will fit its height automatically based
+            on the highest element in that row.
+        col_widths
+            Defines a list of widths for certain columns (left to right). If the list contains `None`, the
+            corresponding column will fit its width automatically based on the widest element in that column.
+        flow_order
+            The order in which submobjects fill the grid. Can be one of the following values:
+            "rd", "dr", "ld", "dl", "ru", "ur", "lu", "ul". ("rd" -> fill rightwards then downwards)
+
+        Returns
+        -------
+        Mobject
+            The mobject.
+
+        NOTES
+        -----
+
+        .. note::
+            If only one of :code:cols and :code:rows is set implicitly, the other one will be chosen big
+            enough to fit all submobjects. If neither is set, they will be chosen to be about the same,
+            tending towards :code:cols > rows (simply because videos are wider than they are high).
+
+        .. note::
+            If both :code:`cell_alignment` and :code:`row_alignments` / :code:`col_alignments` are
+            defined, the latter has higher priority.
+
+
+        Raises
+        ------
+        ValueError
+            If :attr:`rows` and :attr:`cols` are too small to fit all submobjects.
```

Comment:
Doesn't :attr: actually refer to attributes (`self.cols`) instead of keyword arguments? It should probably be changed to reflect that.

```suggestion
            If ``rows`` and ``cols`` are too small to fit all submobjects.
```